web/templates/pages/member_page.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+
+{% load staticfiles %}
+
+{% block title %}{{ user.fullname }}{% endblock %}
+
+{% block content%}
+
+	<div class="row">
+		<div class="member_profile">
+			<div class="col-md-12">
+				<h2>{{ user.fullname }}'s profile</h2>
+			</div>	
+		</div>
+	</div>
+	<div class="row">
+		<div class="col-md-4 col-xs-6">
+			<!--<img  src="{% static 'img/{{user.profile_picture}}' %}" class="img-responsive" alt="Responsive image">-->
+			<img src="../../static/img/{{user.profile_picture}}">
+			<p><strong>GitHub Name:</strong></p>
+			<p>{{ user.githubuser }}</p>
+			<p><strong>Email:</strong></p>
+			<p>{{ user.email }}</p>
+			<p><strong>Web Site:</strong></p>
+			<p>{{ user.website }}</p>
+		</div>
+		<div class="col-md-8 col-xs-6">
+			<p><strong>Join date:</strong>{{ user.date_created|date:"d.m.Y" }}</p>
+			<p><strong>Member's Status:</strong></p>
+			<p>{{ user.activity_status }}</p>
+			<p><strong>Biography:</strong></p>
+			<p>{{ user.bio }}</p>
+		</div>
+		<a href="{% url 'web.views.edituser' user_id=user.id %}">Update profile</a>
```

Comment:
I assume that you couldn't be bothered to name this URL, so you rather introduce this very ugly peace of code `web.views.edituser`.

And I am not sure why are you assigning user.id to user_id variable inside of the `url` tag?

Are you not understanding how urls.py is working?